### ISSUE TITLE
SwiftPM support?

### ISSUE THREAD

**Issue by Andrewangeta** [cashapp/stagehand#13]
Any reason there’s no Swift Package Manager support? Seems like it’d be pretty easy given this is an all swift project.

**Comment by NickEntin** [comment]
Glad to hear you're interested in using Stagehand!

Adding support for SPM is being tracked by #2. I added a bit more detail in there about the blocker for having StagehandTesting support SPM (waiting on a dependency to support it), but we could potentially do it for only Stagehand now. Mainly I was waiting to see if there was interest from the community in using it from SPM.

**Comment by NickEntin** [comment]
Stagehand is now available via SPM! We're still blocked on making StagehandTesting work, which is being tracked by #2.

Closing this issue out. Please reopen or file another issue if you have any trouble getting it to work. Thanks!
